scripts/airdrop.js

```diff
@@ -0,0 +1,54 @@
+const fs = require("fs").promises
+
+const { signAndSend, transactionExistsOnSafeServer } = require("./utils/gnosis_safe_server_interactions")(web3, artifacts)
+const { buildTransferDataFromList } = require("./utils/trading_strategy_helpers")(web3, artifacts)
+const { promptUser } = require("./utils/user_interface_helpers")
+const { default_yargs } = require("./utils/default_yargs")
+const argv = default_yargs
+  .option("masterSafe", {
+    type: "string",
+    describe: "Address of Gnosis Safe transfering funds",
+    demandOption: true,
+  })
+  .option("transferFile", {
+    type: "string",
+    describe: "file name (and path) to the list transfers",
+    demandOption: true,
+  })
+  .option("verify", {
+    type: "boolean",
+    default: false,
+    describe: "Do not actually send transactions, just simulate their submission",
+  })
+  .option("useWei", {
+    type: "boolean",
+    describe: "boolen flag used to indicate if the amounts provided are already in atoms",
+    demandOption: false,
+  }).argv
+
+module.exports = async (callback) => {
+  try {
+    const GnosisSafe = artifacts.require("GnosisSafe")
+    const masterSafe = await GnosisSafe.at(argv.masterSafe)
+
+    const transfers = JSON.parse(await fs.readFile(argv.transferFile, "utf8"))
```

Comment:
For reading a JSON you can just do a `require`

Reply:
Show me!

Reply:
I wouldn't do this: a big difference between `require` and `fs.readFile` is how it interprets relative path strings. The former makes the path relative to the path of the script, while the latter makes the path relative to the working directory of the user when executing the script. SInce this path is inputted by the user, it should be relative to the private working path.

Reply:
In that case, I am going to leave this the way it is, since it works and there is no real problem here.